docs/release-notes/highlights-2.11.0.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[release-highlights-2.11.0]]
+== 2.11.0 release highlights
+
+[float]
+[id="{p}-2110-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 2.11.0 of {n}. Check <<release-notes-2.11.0>> for the full list of changes.
+
+[float]
+[id="{p}-2110-kibana-stack-config-policy"]
+=== Allowing Kibana configuration in stack configuration policies
+
+ECK 2.11.0 supports managing Kibana Authentication Policies via Stack Config Policies (see https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-securing-stack.html[examples]).
```

Comment:
From our internal guidelines:
> Avoid terms that include actions: click, tap, press, see
> Try replacing them with: choose, select, check

```suggestion
Starting with ECK 2.11.0 Stack Config Policies can be used to configure security realms and authentication methods, refer to <<{p}-securing-stack>> for some examples.
```

Reply:
@barkbay  I used a combination of both yours, and @pebrc suggestion here. Let me know if this works for both of you.